src/visitors/Tag.js

```diff
@@ -54,8 +69,19 @@ function getAttributes(
 
       const expr = parseExpression(val === true ? 'true' : val, context);
 
-      if (!mustEscape && (!t.isStringLiteral(expr) || /(\<\>\&)/.test(val))) {
-        throw new Error('Unescaped attributes are not supported in react-pug');
+      if (!mustEscape) {
+        const isStringViaAliases: boolean =
```

Comment:
You shouldn't need to explicitly add the `: boolean` here; flow should infer it from context.